src/lib/wcAutocompleteHandler.ts

```diff
@@ -0,0 +1,36 @@
+import { autocomplete, AutocompleteInstance } from "./autocomplete"
+import { NostoAutocomplete } from "../handlebars/NostoAutocomplete"
+
+type TemplateLib = {
```

Comment:
consider renaming this
something with a Props suffix might be more conventional